sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/timeExpressions.scala

```diff
@@ -233,3 +233,73 @@ object MinuteExpressionBuilder extends ExpressionBuilder {
     }
   }
 }
+
+@ExpressionDescription(
+  usage = """
+    _FUNC_(time_expr) - Returns the hour component of the given time.
+  """,
+  examples = """
+    Examples:
+      > SELECT _FUNC_(TIME'11:35:35.999999');
+       11
+  """,
+  since = "4.1.0",
```

Comment:
I am still confused by this. We registered `HourExpressionBuilder` so this description won't be shown via `DESC FUNCTION EXTENDED hour` right?

Reply:
This won't be shown, and can be removed.

Reply:
@MaxGekk and @HyukjinKwon You suggest to remove below  "@ExpressionDescription" entirely?

```
@ExpressionDescription(
  usage = """
    _FUNC_(time_expr) - Returns the hour component of the given time.
  """,
  examples = """
    Examples:
      > SELECT _FUNC_(TIME'11:35:35.999999');
       11
  """,
  since = "4.1.0",
  group = "datetime_funcs")
  ```

Reply:
Okay Sure @MaxGekk

Reply:
@senthh ? I meant **don't** remove it at `HourExpressionBuilder`, and remote at `HoursOfTime` and at `Hour`.

Reply:
I apologize for the mis-understanding. I will remove "@ExpressionDescription" block  above HoursOfTime and at Hour, but will keep for HourExpressionBuilder